sql/snapshot.sql

```diff
@@ -42,13 +84,13 @@ CREATE INDEX global_snapshot_hash_idx ON public.global_snapshot USING btree (has
 -- DROP TABLE dag_balance_change;
 
 CREATE TABLE dag_balance_change (
-    hash varchar NOT NULL,
-	ordinal int8 NOT NULL,
+	snapshot_hash varchar NOT NULL,
```

Comment:
Even if `snapshot_hash` is the FK to snapshots here, we need `global_ordinal` because we need to be able to query this by address and ordinal like this

```
SELECT balance 
FROM dag_balance_changes 
WHERE global_ordinal < :snapshot_ordinal and address = :address 
LIMIT 1
```